message.py

```diff
@@ -1,4 +1,5 @@
 import smtplib
+import argparse
 
 def send_email(sender, password, receiver, message):
     server = smtplib.SMTP("smtp.gmail.com", 587)
```

Comment:
Create one more file, name it "config.py", put there hardcoded variables "smtp.gmail.com" and 587